time    tick      # Time
float64 height    # Navigation-frame height above take-off point (m)
float64 speed     # Navigation-frame change in height (m/s)